operator/scripts/create_release_charts.sh

```diff
@@ -1,49 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
where's this called? I don't see a reference to it being removed anywhere

Reply:
From operator/scripts/create_assets_gen.sh from gen-charts